hazelcast-spring/src/test/java/com/hazelcast/spring/TestFullApplicationContext.java

```diff
@@ -533,6 +535,8 @@ public void testWanReplicationConfig() {
         assertEquals("10.2.1.1:5701", targetCfg.getEndpoints().get(0));
         assertEquals("10.2.1.2:5701", targetCfg.getEndpoints().get(1));
         assertEquals(wanReplication, wcfg.getTargetClusterConfigs().get(1).getReplicationImplObject());
+        assertEquals(WanAcknowledgeType.ACK_ON_TRANSMIT, wcfg.getTargetClusterConfigs().get(0).getAcknowledgeType());
```

Comment:
Does `getTargetClusterConfigs` reserve the order of config all the time? Otherwise, tests can fail spuriously.